benchmarks/auth/compose_yamls/docker-compose-auth-nodejs.yaml

```diff
@@ -0,0 +1,9 @@
+version: "3.9"
```

Comment:
For all other files you have version 3 do you need it here as well?
Just a question